app/Http/Requests/EventsRequest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+use Illuminate\Foundation\Http\FormRequest;
+
+/**
+ * @codeCoverageIgnore
+ */
+class EventsRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        // only allow updates if the user is logged in
+        return backpack_auth()->check();
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
```

Comment:
I think this is the place to validate the content of the edit/create. For example, we should set the title to be required, because if I leave it empty now it gives an SQL error, because it has no defaul value set.
![screenshot from 2018-09-08 13-57-36](https://user-images.githubusercontent.com/14909670/45253953-77f3ac80-b36f-11e8-8f93-e0af59bc4bdf.png)

Reply:
Rules added